pkg/meta/version.go

```diff
@@ -3,11 +3,11 @@ package meta
 const (
 	// CLIAPIVersion used to communicate with user e.g. longhorn-manager
 	CLIAPIVersion    = 9
-	CLIAPIMinVersion = 3
+	CLIAPIMinVersion = 8
 
 	// ControllerAPIVersion used to communicate with instance-manager
 	ControllerAPIVersion    = 5
-	ControllerAPIMinVersion = 3
+	ControllerAPIMinVersion = 4
```

Comment:
4 was introduced in v1.5, so that means we only allow at least 1.5 IM being used in v1.6.